app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class UserMailer < ApplicationMailer
+  after_action { do_not_deliver unless deliver? }
+
+  private
+
+  def deliver?
+    return false if Settings.mailer.disable_email_to_users
+
+    defined?(@deliver) ? @deliver : true
+  end
+
+  def do_deliver
```

Comment:
Maybe I'm unfamiliar with actionmailer. Does `do_deliver` get invoked somewhere?

Reply:
I just pulled this from another project.  That method is currently unused, but I left it.  Can remove if we want.  

I think I have a better solution though.  One sec.